### PR TITLE
Reduce 3d visualization docs page size from 5.2 to 1.6 MByte

### PR DESCRIPTION
**Pull request by SimonHeybrock**
Should improve load times, and reduce overall github pages size. There are some more pages that may benefit from optimization, but possibly most are just related to the plot resolutions, i.e., not clear if we want to change that.

### CODE REVIEW

File: docs/visualization/plotting/plotting-nd-data.ipynb [hunk 1]
```diff
@@ -191,8 +191,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d.coords['dummy-pos'] = sc.geometry.position(*[d.coords[dim] for dim in ['x', 'y', 'z']])\n",
-    "sc.plot(d, projection='3d', positions='dummy-pos')"
+    "d2 = d['y', :10].copy()\n",
+    "d2.coords['dummy-pos'] = sc.geometry.position(*[d2.coords[dim] for dim in ['x', 'y', 'z']])\n",
+    "sc.plot(d2, projection='3d', positions='dummy-pos')"
```
Comment (nvaytet) [inline docs/visualization/plotting/plotting-nd-data.ipynb:196]: Strange that before these changes, the pixels looked like they had the correct size (1.0)
Screenshot at 2021-12-02 08-02-09
but now they look too small?
Screenshot at 2021-12-02 08-02-39

Comment (SimonHeybrock) [reply docs/visualization/plotting/plotting-nd-data.ipynb:196]: Can't tell why. Maybe just coincidence?

Comment (SimonHeybrock) [reply docs/visualization/plotting/plotting-nd-data.ipynb:196]: Scaling the positions has no influence, so it is probably related to the number of points getting plotted? In any case, probably unrelated to this particular change, i.e., a generic problem(?)?

Comment (nvaytet) [reply docs/visualization/plotting/plotting-nd-data.ipynb:196]: Yes, it's unrelated. See https://github.com/scipp/scipp/pull/2310

**Review by nvaytet — APPROVED** [review]

**Pull request merged**